extensions/amp-form/0.1/amp-form.js

```diff
@@ -473,7 +473,7 @@ export class AmpForm {
             return this.doActionXhr_();
           })
           .then(response => this.handleXhrSubmitSuccess_(
-              /* !../../../src/service/xhr-impl.FetchResponse */ response),
+              /* !../../../src/utils/xhr-utils.FetchResponse */ response),
```

Comment:
`/* @type {...} */`

Reply:
nope. oddly..but they were failing for a CL i was modifying that edited some of these files.

Reply:
done